src/pages/project/project_wizard/ProjectDetailsForm.tsx

```diff
@@ -1,10 +1,95 @@
+  const { t } = useTranslation();
+  const { values, setFieldValue, setFieldTouched, touched, errors } = useFormikContext<SaveCristinProject>();
+  const thisIsRekProject = isRekProject(project);
 
-export const ProjectDetailsForm = () => {
   return (
     <ErrorBoundary>
-      <Box sx={{ bgcolor: 'secondary.light', padding: '1.75rem 1.25rem' }}>test 2</Box>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+        <FormBox>
+          <Typography variant="h2">{t('project.coordinating_institution')}</Typography>
+          <Field name={ProjectFieldName.CoordinatingInstitution}>
+            {({ field }: FieldProps<ProjectOrganization>) => (
+              <OrganizationSearchField
+                disabled={thisIsRekProject}
+                label={t('project.coordinating_institution')}
+                onChange={(selectedInstitution) => {
+                  const selectedCoordinatingInstitution: ProjectOrganization = {
+                    type: 'Organization',
+                    id: selectedInstitution?.id ?? '',
+                    labels: selectedInstitution?.labels ?? {},
+                  };
+                  setFieldValue(field.name, selectedCoordinatingInstitution);
+                }}
+                errorMessage={
+                  touched.coordinatingInstitution?.id && !!errors.coordinatingInstitution?.id
+                    ? errors.coordinatingInstitution?.id
+                    : undefined
+                }
+                fieldInputProps={{
+                  ...field,
+                  onBlur: () => {
+                    setFieldTouched(`${field.name}.id`);
+                  },
+                }}
+                selectedValue={field.value}
+                customDataTestId={dataTestId.registrationWizard.description.projectForm.coordinatingInstitutionField}
+              />
+            )}
+          </Field>
+        </FormBox>
+        <FormBox>
+          <Typography variant="h2" sx={{ pb: '0.25rem' }}>
+            {t('project.form.category')}
+          </Typography>
+          <Field name={ProjectFieldName.Categories}>
+            {({ field, form: { setFieldValue } }: FieldProps<TypedLabel[]>) => (
+              <Autocomplete
+                options={cristinCategories}
+                multiple
+                data-testid={dataTestId.registrationWizard.description.projectForm.projectCategoryField}
+                getOptionLabel={(option) => getLanguageString(option.label)}
+                isOptionEqualToValue={(option, value) => option.type === value.type}
+                value={field.value}
+                onChange={(_, value) => setFieldValue(field.name, value)}
+                renderInput={(params) => (
+                  <TextField {...params} variant="filled" label={t('project.project_category')} />
+                )}
+              />
+            )}
+          </Field>
+        </FormBox>
+        <FormBox sx={{ gap: '0.5rem' }}>
```

Comment:
gap-en her gjør at denne boksen får mindre gap etter heading enn de andre boksene. Jeg ser for meg at vi kommer til å ta i bruk `FormBox` i registration wizard også, nå som skissene har samme gruppering av input-felter der.

Lurer derfor på om det kan være lurt at vi fjerner `display: "flex"` fra boksen, sånn at det bare blir en dum boks som står for en gruppering som vi selv bestemmer hvordan skal vise innholdet alle plasser den brukes..?

Det vil føre til at vi enten må sette feks `gutterBottom`, `mb`, e.l. på typography for å få gaps, eller ev. definerer en flex i sx inline. Om vi overstyrer gap som her vil det føre til ulik spacing for heading i boksene, som kan se litt rart ut

(dette er bare meg som tenker høyt, så kan du selv vurdere om du vil gjøre noe med det nå, eller senere, eller bare ignorere det :D)

![bilde](https://github.com/user-attachments/assets/dace8068-d0cf-45d7-b635-a388d31c7270)

Reply:
Jeg tenker de fleste vil bruke flex, så vi kan heller gjøre det mulig å overstyre for å ta det bort (altså ta imot sx-props), så slipper vi å sende inn sx med flex hver eneste gang vi bruker den. gap i flex gir jo da også en standard-avstand, og så kan vi jo vurdere å gjøre det til en komponent der vi sender inn overskrift som props om vi vil ha større kontroll etterhvert.